input/docs/report-formats/generic/templates/htmldatatable.md

```diff
@@ -15,4 +15,6 @@ Template for a HTML report containing a rich data table view with sorting and se
 
 ## Screenshots
 
-![HTML Data Table](htmldatatable01.png "HTML Data Table")
\ No newline at end of file
+![HTML Data Table](htmldatatable01.png "HTML Data Table")
+
+[Vanilla-DataTables]: https://github.com/Mobius1/Vanilla-DataTables
```

Comment:
Add missing trailing **/**

Reply:
GitHub links also without a trailing slash, so I prefer to keep it like this